hs/src/Reach/AST/DLBase.hs

```diff
@@ -193,19 +197,26 @@ instance Pretty DLInit where
 
 data DLConstant
   = DLC_UInt_max
-  deriving (Eq, Generic, Show, Ord)
+  | DLC_Zero_addr
```

Comment:
It's not really a zero address... it is a zero token, which would be `DLC_Token_zero`